docs/UserGuide.md

```diff
@@ -75,26 +75,46 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a volunteer: `add` [COMING SOON]
```

Comment:
should be `volunteer add` or `v add` since we decided on this to distinguish volunteers from events